spacy/tests/lang/es/test_noun_chunks.py

```diff
@@ -1,6 +1,148 @@
+from spacy.tokens import Doc
 import pytest
 
 
+# fmt: off
+@pytest.mark.parametrize(
+    "words,heads,deps,pos,chunk_offsets",
+    [
+        # un gato -> "un gato"
+        (
+            ["un", "gato"],
+            [1, 1],
+            ["det", "ROOT"],
+            ["DET", "NOUN"],
+            [(0, 2)],
+        ),
+        # la camisa negra -> "la camisa negra"
+        (
+            ["la", "camisa", "negra"],
+            [1, 1, 1],
+            ["det", "ROOT", "amod"],
+            ["DET", "NOUN", "ADJ"],
+            [(0, 3)],
+        ),
+        # un lindo gatito -> "un lindo gatito"
+        (
+            ["Un", "lindo", "gatito"],
+            [2, 2, 2],
+            ["det", "amod", "ROOT"],
+            ["DET", "ADJ", "NOUN"],
+            [(0,3)]
+        ),
+        # una chica hermosa e inteligente -> una chica hermosa e inteligente
```

Comment:
So I tried with noun chunks that have modifiers on both sides such as "the fabulous brown cat" and I think the code is not getting those as full chunks. I get an assertion error if I try the following: 
```
        # el fabuloso gato pardo -> "el fabuloso gato pardo"
        (
            ["el", "fabuloso", "gato", "pardo"],
            [3, 3, 3, 3],
            ["det", "amod", "ROOT", "amod"],
            ["DET", "PROPN", "PROPN", "PROPN"],
            [(0, 3)],
        ),
```
Fails with: assert [(2, 3)] == [(0, 3)]

Reply:
> So I tried with noun chunks that have modifiers on both sides such as "the fabulous brown cat" and I think the code is not getting those as full chunks. I get an assertion error if I try the following:
> 
> ```
>         # el fabuloso gato pardo -> "el fabuloso gato pardo"
>         (
>             ["el", "fabuloso", "gato", "pardo"],
>             [3, 3, 3, 3],
>             ["det", "amod", "ROOT", "amod"],
>             ["DET", "PROPN", "PROPN", "PROPN"],
>             [(0, 3)],
>         ),
> ```
> 
> Fails with: assert [(2, 3)] == [(0, 3)]

Argh, can we try changing the -3th line to:
```
["DET", "ADJ', "PROPN", "ADJ"]
```
pos tags doesn't seem correct to me :slightly_smiling_face:

Reply:
Ah yes! I was using the tags and parse structure from one of the small language models in Spanish. This is better :)

Reply:
I added this case to the unit tests, thanks for that!